Missing dependencies from package.json

Missing dependencies such mkdirp

I'm getting this error

internal/modules/cjs/loader.js:969
  throw err;
  ^

Error: Cannot find module 'mkdirp'
Require stack:
- C:\CostisWork\NodeJS\autoTranslate\node_modules\fraud\dist\index.js
- C:\CostisWork\NodeJS\autoTranslate\node_modules\auto-i18n\dist\index.js
- C:\CostisWork\NodeJS\autoTranslate\index.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:966:15)
    at Function.Module._load (internal/modules/cjs/loader.js:842:27)
    at Module.require (internal/modules/cjs/loader.js:1026:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at Object.<anonymous> (C:\CostisWork\NodeJS\autoTranslate\node_modules\fraud\dist\index.js:16:14)
    at Module._compile (internal/modules/cjs/loader.js:1138:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1158:10)
    at Module.load (internal/modules/cjs/loader.js:986:32)
    at Function.Module._load (internal/modules/cjs/loader.js:879:14)
    at Module.require (internal/modules/cjs/loader.js:1026:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    'C:\\CostisWork\\NodeJS\\autoTranslate\\node_modules\\fraud\\dist\\index.js',
    'C:\\CostisWork\\NodeJS\\autoTranslate\\node_modules\\auto-i18n\\dist\\index.js',
    'C:\\CostisWork\\NodeJS\\autoTranslate\\index.js'
  ]
}

--- a/package.json
+++ b/package.json
@@ -27,7 +27,6 @@
     "@types/node": "^14.11.8",
     "concurrently": "^5.3.0",
     "coveralls": "^3.1.0",
-    "dotenv": "^8.2.0",
     "jest": "^26.5.3",
     "onchange": "^7.0.2",
     "prettier": "^2.1.2",
@@ -54,7 +53,9 @@
   "dependencies": {
     "@google-cloud/translate": "^4.1.3",
     "dot-object": "^2.1.4",
+    "dotenv": "^8.2.0",
     "fraud": "^5.2.0",
-    "fs-extra": "^9.0.1"
+    "fs-extra": "^9.0.1",
+    "mkdirp": "^0.5.6"
   }
 }

--- a/tests/package.test.ts
+++ b/tests/package.test.ts
@@ -1,0 +1,7 @@
+const packageJson = require("../package.json");
+
+test("declares modules needed by production installs", () => {
+  expect(packageJson.dependencies).toHaveProperty("dotenv");
+  expect(packageJson.dependencies).toHaveProperty("mkdirp");
+  expect(packageJson.devDependencies || {}).not.toHaveProperty("dotenv");
+});

